modules/ROOT/pages/migrate-ah.adoc

```diff
@@ -0,0 +1,42 @@
+:experimental:
+
+= Migrating from Fedora Atomic Host (FAH) to Fedora CoreOS (FCOS)
+
+== Overview
+
+https://www.projectatomic.io/[Fedora Atomic Host] was a system to deploy applications in containers. Existing FAH users are encouraged to migrate to FCOS, as the project has reached its end-of-life.
+
+FAH used `cloud-init` for provisioning, which required users to provide a `cloud-config` file as userdata for configuration of the instance. Since FCOS Ignition and `cloud-init` are different and have overlapping feature sets, it is not trivial to convert `cloud-init` files to Ignition. Currently, there is no tool for this conversion, so you must manually convert `cloud-init` configs to Fedora CoreOS configs. Refer to xref:fcct-config.adoc[FCCT Specification] for an explanation of the available configuration options.
+
+== Converting `cloud-init` and `cloud-config` userdata
+
+The following examples show the difference between FAH userdata and user configuration with FCC.
+
+.Example of FAH userdata file:
+----
+#cloud-config
+password: atomic
+ssh_pwauth: True
+chpasswd: { expire: False }
+
+ssh_authorized_keys:
+  - ssh-rsa ...
+----
+
+Manually configure with a xref:ign-passwd.adoc[`passwd`] node in the FCC file:
```

Comment:
Probably should say: 'This can be manually translated into a xref:ign-passwd.adoc[`passwd`] node in the FCC file:'

Reply:
followup in https://github.com/coreos/fedora-coreos-docs/pull/39